server/src/tenant/schema-generation/graphql-types/object.graphql-type.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLID,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+import { FieldMetadata } from 'src/tenant/metadata/field-metadata/field-metadata.entity';
+import { ObjectMetadata } from 'src/tenant/metadata/object-metadata/object-metadata.entity';
+import { pascalCase } from 'src/utils/pascal-case';
+
+/**
+ * Map the column type to its corresponding GraphQL type.
+ * @param columnType Type of the column in the database.
```

Comment:
I would precise "type from field-metadata" column or something, otherwise it looks like it's the real type of the column in DB